src/handbook/development/releases/planning.md

```diff
@@ -49,8 +49,7 @@ Issues are the building blocks of planning activities, helping the team to manag
    a user should be able to do and identify the value that brings to the user.
    A story should be deliverable in a single release.
 
- - **Task**: a piece of work that isn't necessarily tied to a specific Epic or Story.
-   For example, items related to technical debt or house-keeping chores.
+ - **Task**: If we consider a Story as a user-centric view of a feature, the underlying tasks are the engineering-centric view of the same feature. This may also be a  piece of work that isn't necessarily tied to a specific Epic or Story. For example, items related to technical debt or house-keeping chores.
```

Comment:
```suggestion
 - **Task**: If we consider a Story as a user-centric view of a feature, the underlying tasks are the engineering-centric view of the same feature. This may also be a piece of work that isn't necessarily tied to a specific Epic or Story. For example, items related to technical debt or house-keeping chores.
```